docs/file-formats/xml-data-file.md

```diff
@@ -986,15 +986,15 @@ shortcut, so that the XML file can be validated more precisely.
 `<video-segment>` and `<audio-segment>` represent sections of an audio/video file. 
 They can have the following predefined properties:
 
-| prop name          | cardinality | prop type     | encoding |
-| ------------------ | ----------- | ------------- | -------- |
-| `isSegmentOf`      | 1           | resptr-prop   |          |
-| `hasSegmentBounds` | 1           | interval-prop |          |
-| `hasTitle`         | 0-1         | text-prop     | utf8     |
-| `hasDescription`   | 0-n         | text-prop     | xml      |
-| `hasComment`       | 0-n         | text-prop     | xml      |
-| `hasKeyword`       | 0-n         | text-prop     | utf8     |
-| `relatesTo`        | 0-n         | resptr-prop   |          |
+| prop name          | cardinality | prop type     | text type   |
+| ------------------ | ----------- | ------------- | ----------- |
+| `isSegmentOf`      | 1           | resptr-prop   |             |
+| `hasSegmentBounds` | 1           | interval-prop |             |
+| `hasTitle`         | 0-1         | text-prop     | unformatted |
```

Comment:
Perhaps write the encoding in the brackets. Till we made that distinction without them having to enter the encoding, I would mention it in the documentation.

Reply:
Here, they don't have to enter the encoding, so it's better to omit the encoding